TouchID.android.js

```diff
@@ -2,17 +2,18 @@ import { NativeModules, processColor } from 'react-native';
 import { androidApiErrorMap, androidModuleErrorMap } from './data/errors';
 import { getError, TouchIDError, TouchIDUnifiedError } from './errors';
 const NativeTouchID = NativeModules.FingerprintAuth;
+import { DeviceEventEmitter } from 'react-native';
```

Comment:
In the first line you can find the imports from `react-native` module, should be joined.